### DISCUSSION TITLE
NTV News

### DISCUSSION THREAD

**Discussion by greenmr** [iptv-org/community#858] [Show and tell]
### Channel ID

NTV.ca

### Stream URL

https://2-fss-1.streamhoster.com/pl_122/201748-1431018-1/chunklist.m3u8

### Notes

This stream url has been stable for well over a year. It was extracted from the NTV web player. I have tested this stream with Surfshark from multiple countries outside Canada.

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by AntiPontifex** [comment]
Thank you for the stream: it will be included in the next release.

**Comment by freearhey** [comment]
https://2-fss-1.streamhoster.com/pl_122/201748-1431018-1/chunklist.m3u8 (offline)
<img width="540" alt="image" src="https://user-images.githubusercontent.com/7253922/233796719-7fd5e3c6-0f12-485f-acb0-420c7800e071.png">

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19598
